### DISCUSSION TITLE
Hosting Frontend

### DISCUSSION THREAD

**Discussion by yevgenybulochnik** [coderxio/sagerx#102] [General]
# Problem Statement
We need to decide what we are going to use for a front end. Part of this will hinge on if we want to collect information from users right off the bat.

Questions:
- Do we want user logins?
- Do we care about SEO?
- Do we want to have other content beyond flat files and an API?
- If we create the api part later should that actually drive user auth? 

# Criteria for Success
No measurable outcome yet, open discussion about pros and cons

# Additional Information

Possible front ends:
- [nextjs](https://nextjs.org/)

**Comment by yevgenybulochnik** [comment]
I have been toying around with an idea of creating a MVP select beta (or maybe its really alpha). Locally host something simple and ask select members of coderx to take a look and provide feedback. Make adjustments off that feedback.

**Comment by jrlegrand** [comment]
Yes I like that idea.  We could even divide and conquer as we pilot this and iterate.  Kent keeps adding and refining data sources/staging tables/views, Joey works on front end/strategy/general other issues, Yevgeny works on backend API/flatfile generation/etc.  

We could even post a signup to see if other people on LinkedIn (etc) are interested and then select the ones we want to give access to.
